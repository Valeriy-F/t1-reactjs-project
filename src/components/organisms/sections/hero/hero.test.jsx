import { render, screen } from "@testing-library/react";

import Hero from "./hero";

describe("Hero section tests", () => {
  test("Hero section snapshots", () => {
    render(<Hero />);

    const quizComponent = screen.queryByTestId("hero");

    expect(quizComponent).toBeInTheDocument();
    expect(quizComponent).toMatchSnapshot();
  });
});
