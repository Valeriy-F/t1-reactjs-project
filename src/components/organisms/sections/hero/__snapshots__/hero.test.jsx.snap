// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero section tests Hero section snapshots 1`] = `
<div
  class="content-container"
  data-testid="hero"
>
  <div
    class="title"
  >
    <h1
      class="variant-h1 color-secondary"
    >
      Any products from famous brands with worldwide delivery
    </h1>
  </div>
  <div
    class="subtitle"
  >
    <p
      class="variant-text_bold color-secondary"
    >
      We sell smartphones, laptops, clothes, shoes and many other products at low prices
    </p>
  </div>
  <div
    class="actions"
  >
    <a
      aria-label="Go to shopping"
      class="base variant-button size-lg color-secondary "
      href="/products"
    >
      Go to shopping
    </a>
  </div>
</div>
`;
